sql/database.sql

```diff
@@ -1281,10 +1281,14 @@ CREATE TABLE `form_encounter` (
   `stmt_count`        int  NOT NULL DEFAULT 0,
   `provider_id` INT(11) DEFAULT '0' COMMENT 'default and main provider for this visit',
   `supervisor_id` INT(11) DEFAULT '0' COMMENT 'supervising provider, if any, for this visit',
+  `ordering_physician` INT(11) DEFAULT '0' COMMENT 'ordering provider , if any, for this visit',
+  `referring_physician` INT(11) DEFAULT '0' COMMENT 'referring provider, if any, for this visit',
+  `contract_physician` INT(11) DEFAULT '0' COMMENT 'contract provider, if any, for this visit',  
   `invoice_refno` varchar(31) NOT NULL DEFAULT '',
   `referral_source` varchar(31) NOT NULL DEFAULT '',
   `billing_facility` INT(11) NOT NULL DEFAULT 0,
   `external_id` VARCHAR(20) DEFAULT NULL,
+  `pos_code` tinyint(4) default NULL,
```

Comment:
The facility code should come from "service_facility" and not a separate POS code.  That means if you have an error in the facility info, you can change that, and all the encounters are fixed.